share/spice/frequency_allocation/frequency_allocation.handlebars

```diff
@@ -0,0 +1,4 @@
+<span>{{name}}</span>
```

Comment:
Doesn't look like this nor the CSS file above is being used. If not, they can be removed.